terratest/test/k8gb_basic_roundrobin_test.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2021 Absa Group Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
+*/
+package test
+
+import (
+	"fmt"
+	"path/filepath"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Basic k8gb deployment test that is verifying that associated ingress is getting created
+// Relies on two local clusters deployed by `$make deploy-two-local-clusters`
+func TestK8gbBasicRoundRobinExample(t *testing.T) {
+	t.Parallel()
+	const host = "roundrobin-test.cloud.example.com"
+	const gslbName = "roundrobin-test-gslb"
+	var waitForLocalGSLB = func(port int, expectedResult []string) (output []string, err error) {
```

Comment:
Cool stuff on reusing existing helper. Still probably makes sense to put it to helpers as well so `waitForLocalGSLB` will be available in other tests?

Reply:
@ytsarev amended